commands/pull_request.go

```diff
@@ -55,6 +56,7 @@ func init() {
 	cmdPullRequest.Flag.BoolVarP(&flagPullRequestForce, "force", "f", false, "FORCE")
 	cmdPullRequest.Flag.StringVarP(&flagPullRequestFile, "file", "F", "", "FILE")
 	cmdPullRequest.Flag.StringVarP(&flagPullRequestAssignee, "assign", "a", "", "USER")
+	cmdPullRequest.Flag.BoolVarP(&flagPullRequestEditor, "editor", "e", false, "EDITOR")
```

Comment:
For parity with `git commit -e`, I think the long name for this flag should be `--edit`, not `--editor`.